pkg/cassandra/config/config.go

```diff
@@ -58,6 +58,18 @@ type Schema struct {
 	// while connecting to the Cassandra Cluster. This is useful for connecting to clusters, like Azure Cosmos DB,
 	// that do not support SnappyCompression.
 	DisableCompression bool `mapstructure:"disable_compression"`
+	// CreateSchema tells if the schema ahould be created during session initialization based on the configs provided
+	CreateSchema bool `mapstructure:"create" valid:"optional"`
+	// Datacenter is the name for network topology
+	Datacenter string `mapstructure:"datacenter" valid:"optional"`
+	// TraceTTL is Time To Live (TTL) for the trace data. Should at least be 1 second
+	TraceTTL time.Duration `mapstructure:"trace_ttl" valid:"optional,cassandraTTLValidation"`
+	// DependenciesTTL is Time To Live (TTL) for dependencies data. Should at least be 1 second
+	DependenciesTTL time.Duration `mapstructure:"dependencies_ttl" valid:"optional,cassandraTTLValidation"`
+	// Replication factor for the db
+	ReplicationFactor int `mapstructure:"replication_factor" valid:"optional"`
+	// CompactionWindow of format tells the compaction window of the db. Should atleast be 1 minute
```

Comment:
why are we checking 1sec / 1min? I don't think these are in any sense practical limits - compaction can take hours, you don't do it every 1min. I would only validate that the value is not negative, otherwise it's up to the user.

Reply:
Same goes for `default_time_to_live` i.e., `TraceTTL` and `DependenciesTTL`, where the minimum value it accepts is 1 second.